meow/templates/scheduler/edit.html

```diff
@@ -47,20 +47,18 @@
     $('#send_now').click(function(e) {
         e.preventDefault();
         console.log("send now button was clicked");
-        var postId = "{{ post.id }}";
-        console.log({{ post.id }});
         $.ajax({
-            url: "/api/send-now/" + postId + "/",
+            url: "/api/send-now/" + {{ post.id }} + "/",
             method: "POST",
             headers: {'X-CSRFToken': '{{ csrf_token }}'},
-            data: postId,
+            data: "{{ post.id }}",
             dataType: "json"
         }).done(function(response) {
             console.log(response.id + " " + response.name);
         }).fail(function (error) {
             console.log(error);
         });
-        //window.location.href = '../api/send-now/';
+        window.location.href = '/';
```

Comment:
This needs to be moved to the done callback above. There's a chance the request is not sent out fully in this case since this is executed immediately after the promise is registered. Might also be nice to show a very rudimentary prompt (alert?) If there's an error